GruntFile.js

```diff
@@ -291,6 +291,17 @@ module.exports = function (grunt) {
             },
          },
          files: ['build/package.json']
+      },
+
+      connect: {
+         sandbox: {
+            options: {
+               port: '3001',
+               useAvailablePort: true,
+               keepalive: true,
+               base: './sandbox'
+            }
```

Comment:
Nice! Will this keep us from having to spin up a local server every type we want to test sandbox???? 🥇

Reply:
YEE 💯

Reply:
And can use to do functional tests if needed.